### ISSUE TITLE
50% accuracy with alexnet after 300 epochs

### ISSUE THREAD

**Issue by nachiket008** [wangyirui/AD_Prediction#20]
Hi @wangyirui ,
I downloaded the data from ADNI site and also skull stipped it. Now when I run the code with main_alexnet.py I am getting very low accuracy. The accuracy is only around 51% which is far away from what I can see in the results. I am not sure why this is happening. Previously when the data wasn't skull stripped I thought that was the reason. But even after skull stripping I am getting low accuracy. I can see in other comments where someone else achieved 80% accuracy too. Can you please guide me on this. 
Thanks for your help. :)

**Comment by wangyirui** [comment]
Hi,

Thanks for your interest in my project. Did you do registration (template
mapping) before running the training code? The registration will map each
MRI brain image to a standard template. You can check my final report in
the repo to see the details.

Best,
Yirui

nachiket008 <notifications@github.com> 于2019年3月26日周二 上午5:48写道：

**Comment by nachiket008** [comment]
Hi @wangyirui ,
Thanks for the quick response and advice. I had not performed registration. I had some trouble with SPM. I had one more query... Please can you advise on this.
For main_alexnet.py which data did you use? Was it grey matter , white matter or both. In the Image directory where the training and testing files are stored.. which data is present in those files.

**Comment by FaRe3** [comment]
> Hi @wangyirui ,
> Thanks for the quick response and advice. I had not performed registration. I had some trouble with SPM. I had one more query... Please can you advise on this.
> For main_alexnet.py which data did you use? Was it grey matter , white matter or both. In the Image directory where the training and testing files are stored.. which data is present in those files.

Did this code work for you. Please let me know
Thank you
